### ISSUE TITLE
[Issue]errors when using >color in appilcation launcher

### ISSUE THREAD

**Issue by kittenvr** [end-4/dots-hyprland#857]
- I have read the Usage and Troubleshooting pages of the wiki: https://end-4.github.io/dots-hyprland-wiki/en/i-i/03troubleshooting/
- I have made sure that both my config and system packages are up to date
- Linux distro: 

---
## The issue

When I use >color in the application launcher my screen goes weird(see screenshot)

![20241009_212924](https://github.com/user-attachments/assets/31805a69-58df-452b-87ea-f92881a0461b)
![20241009_212936](https://github.com/user-attachments/assets/89f024c6-7afd-4645-a0d7-bc91064f80a6)

I'm on cachyos hyprland

<details>
<summary>Logs</summary>

```plain
https://www.logpasta.com/paste/2b980486-3fbd-4590-be9d-4ae1ce1e7fcd
```

</details>

**Comment by l4v3nx** [comment]
apply any wallpaper with ctrl+super+t

**Comment by Ranger-Dj** [comment]
thanks bro i got the same issue

**Comment by kittenvr** [comment]
> apply any wallpaper with ctrl+super+t

That didn't work

**Comment by nils-affentranger** [comment]
+1, I have exactly the same issue. Glad this isn't just me

**Comment by end-4** [comment]
can you send the output of this?
```sh
~/.config/ags/scripts/color_generation/switchwall.sh
```

**Comment by thnhmai38** [comment]
I got the same when I check this box
![20241018_113830.jpg](https://github.com/user-attachments/assets/618d8461-104a-423c-bf94-e41b89441414)

**Comment by kittenvr** [comment]
> I got the same when I check this box
> ![20241018_113830.jpg](https://github.com/user-attachments/assets/618d8461-104a-423c-bf94-e41b89441414)
> 
> Update: Fixed when change wallpaper (At first there was no wallpaper)

Yeah I also have that issue but it doesn't get fixed on wallpaper change

**Comment by kittenvr** [comment]
> > apply any wallpaper with ctrl+super+t
> 
> This works for me 🙂

not for me

**Comment by KofTD** [comment]
If changing a wallpaper doesn't help you, try checking HOME/.config/ags/scripts/color_generation/colorgen.sh

In my case, this script doesn't see “materialyoucolor” module, but it has been installed. I reinstalled it and it works.

**Comment by clsty** [comment]
It should be fine now after #1060. Just `git pull` for the latest version and `./install.sh` again, then switch wallpaper again.

Closing.
